### PR TITLE
improve documentation

### PR DESCRIPTION
**Pull request by pinage404**
## Summary

I just discovered DevBox

I read the whole documentation

I found some errors

I did a big PR with anrelated content, if you disagree with some of the commits, i can split in several PR

## How was it tested?

It's just markdown content, i haven't tested it

### CODE REVIEW

File: docs/app/docs/guides/pinning_packages.md [hunk 1]
```diff
@@ -17,21 +17,21 @@ Because the repository changes frequently, and new releases of Nixpkgs infrequen
...
-16.15.0, 16.14.0, 16.13.1, 16.13.0, 16.8.0, 16.4.0, 16.0.0, 15.14.0, 15.10.0, 15.5.0, 15.0.1,
-14.18.1, 14.18.0, 14.17.5, 14.17.1, 14.16.1, 14.16.0, 14.15.3, 14.15.0, 14.9.0, 14.4.0, 13.14.0,
-12.22.12, 12.22.10, 12.22.8, 12.22.7, 12.22.5, 12.22.1, 12.21.0, 12.20.0, 12.19.0, 12.18.3,
-12.18.1, 10.24.1, 10.24.0, 10.23.0, 10.22.0, 10.21.0)
-* nodejs_16 (16.20.0)
-* nodejs_18 (18.16.0)
-* nodejs_19 (19.9.0)
-* nodejs_20 (20.0.0)
-...
+$ devbox search python
```
Comment (LucilleH) [inline docs/app/docs/guides/pinning_packages.md:20]: @Lagoja did you intend to include the nodejs results before, or was it supposed to be python?

**Review by LucilleH — APPROVED** [review]
Thank you!!

**Comment by Akiyamka** [comment]
I wonder, why nodejs packages in examples was replaces with python

File: docs/app/docs/guides/pinning_packages.md (hunk 1 above)
Comment (Lagoja) [reply docs/app/docs/guides/pinning_packages.md:20]: I think it should be nodejs for this example, since the results look cleaner

**Comment by Lagoja** [comment]
Awesome, looks good to land! Thanks again @pinage404!

**Pull request merged**